.github/workflows/Nightly.yaml

```diff
@@ -0,0 +1,161 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron:  '0 5 * * *'
+
+concurrency:
```

Comment:
After reading about `concurrency` I don't understand why you're using it here, so remove it or add a comment explaining